uiowa/uiowa
# Feature layout block restrictions

Relates to #955

# Updates
Removes the ability to add the page title hero and the spacer anywhere. Restricts banner/hero in multi-column layouts.

# To test
* `composer install` - There is an update to `lb_direct_add` module that is necessary for restricted options to not show up.
* View the instructions at #1932 about how to pull down/install various layout builder test sites.
* Visit a page and click the layout tab (e.g. https://gradcogscilang.local.drupal.uiowa.edu/node/1/layout)
* Add 1, 2, 3, and 4 column layouts and check the following:
  * Banner/Hero should not be available on 2-, 3-, and 4-column layouts.
  * Page title hero and separator/spacer should not be available.

## Policy site
Policy splits its entity_display_view for pages, so I copied these changes to that split as well. @mark-bennett-uiowa, please feel free to give feedback about this.
